### PR TITLE
Allow view selection to function with JS disabled

### PR DESCRIPTION
**Pull request by borkweb**
See: https://central.tri.be/issues/27982

### CODE REVIEW

File: src/Tribe/Main.php [hunk 1]
```diff
@@ -707,12 +707,38 @@ public function initOptions() {
...
 		/**
+		 * handles tribe bar post submissions
+		 *
+		 * @param array $postdata Data from $_POST
+		 */
+		public function handle_submit_bar_redirect( $postdata ) {
+			$url = $postdata['tribe-bar-view'];
+
+			foreach ( $postdata as $key => $value ) {
+				if ( 'submit-bar' === $key || 'tribe-bar-view' === $key ) {
```
Comment (bordoni) [inline src/Tribe/Main.php]: Why not `in_array`?

Comment (borkweb) [inline src/Tribe/Main.php]: It takes more typing if there are only two values :)

Comment (bordoni) [inline src/Tribe/Main.php]: Right... Ok...

**Comment by bordoni** [comment]
:wine_glass: :snowman:

**Pull request merged**